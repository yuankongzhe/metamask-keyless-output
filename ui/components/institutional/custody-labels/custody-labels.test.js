import React from 'react';
import { render } from '@testing-library/react';
import CustodyLabels from './custody-labels';

describe('CustodyLabels Component', () => {
  it('should render correctly', () => {
    const props = {
      labels: [{ key: 'testKey', value: 'value' }],
      index: 'index',
      hideNetwork: 'true',
    };

    const { container } = render(<CustodyLabels {...props} />);

    expect(container).toMatchSnapshot();
  });
});
