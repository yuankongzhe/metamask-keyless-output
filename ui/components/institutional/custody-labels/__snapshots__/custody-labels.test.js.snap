// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustodyLabels Component should render correctly 1`] = `
<div>
  <label
    class="mm-box mm-text mm-label mm-label--html-for mm-text--body-md mm-text--font-weight-medium mm-box--display-flex mm-box--flex-direction-row mm-box--align-items-center mm-box--color-text-default"
    for="address-index"
  >
    <p
      class="mm-box mm-text custody-label mm-text--body-xs mm-text--font-weight-normal mm-text--text-transform-capitalize mm-box--margin-top-1 mm-box--margin-right-1 mm-box--margin-bottom-2 mm-box--padding-top-1 mm-box--padding-right-2 mm-box--padding-bottom-1 mm-box--padding-left-2 mm-box--color-text-default mm-box--background-color-background-alternative mm-box--rounded-sm"
    >
      value
    </p>
  </label>
</div>
`;
